src/components/previewers/ImagePreviewer.jsx

```
@@ -95,14 +102,15 @@ const ImagePreviewer = ({
-              onClick={handleCopy}
-              className={`action-button hover:${textColor}`}
-              aria-label="Copy Link"
-            >
-              <FaCopy className="h-6 w-6" /> Copy Link
-            </button>
+            {!isEmoji && (
+              <button
+                onClick={handleCopy}
+                className={`action-button hover:${textColor}`}
+                aria-label="Copy Link"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)